config/locales/views/users/en.yml

```
@@ -91,4 +91,5 @@ en:
       see_all : "See All"
       fork_notification: "%{user} forked your Project %{project}"
       star_notification: "%{user} starred your Project %{project}"
-      new_assignment_notification: "You have been assigned a new assignment '%{assignment_name}'"
\ No newline at end of file
+      new_assignment_notification: "You have been assigned a new assignment '%{assignment_name}'"
+      clear_all: "Clear All"
```

```suggestion
      clear_all: "Clear All"
      clear_all_confirmation: "Are you sure you want to clear all notifications?"
```
You also need to add the hindi translation in `hi.yml`

Use these text:
```
      clear_all: "सभी नोटिफिकेशन हटाएं"
      clear_all_confirmation: "क्या आप वाकई सभी नोटिफिकेशन हटाना करना चाहते हैं?"
```